src/org/zaproxy/zap/extension/api/ApiGeneratorUtils.java

```diff
@@ -101,6 +102,8 @@
 
 		imps.add(new ScriptAPI(null));
 
+		imps.add(new StatsAPI());
```

Comment:
It should add the `StatsParam` to `StatsAPI` before, to include the options. Or was that on purpose?